run-profile-partitions

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+
+# Assume that assignments have been generated into ../zeth/build
+
+ASSIGNMENTS_DIR=../zeth/build
+ZETH_TOOL=../zeth/build/zeth_tool/zeth-tool
+STORE_HOST=ec2-user@ip-172-31-27-253
```

Comment:
Why do we have an IP here?

Reply:
I've removed these hard-coded paths and added checks for env vars.